docs/source/index.rst

```diff
@@ -17,18 +17,18 @@ The easiest way to build Machine Learning APIs
 
 *Multi-framework / High-performance / Easy to learn / Production ready*
 
-What does BentoML do?
----------------------
+BentoML makes moving trained ML models to production easy:
```

Comment:
Is this line too large for a title? Could you play with it see what font size looks better?